src/test/java/uk/gov/ons/census/casesvc/service/CaseServiceTest.java

```diff
@@ -111,6 +114,115 @@ public void testSaveCaseSample() {
     assertThat(savedCase.getCeActualResponses()).isEqualTo(0);
     assertThat(savedCase.getCaseType()).isEqualTo(TEST_ADDRESS_TYPE);
     assertThat(savedCase.isHandDelivery()).isFalse();
+    assertThat(savedCase.getMetadata().getSecureEstablishment()).isFalse();
+  }
+
+  @Test
+  public void testSaveCaseSampleCE() {
```

Comment:
what does this test add over testSaveCaseSampleCESecureFalse? both test it starts of 0 in the input and ends up being 'saved' as False?

Reply:
Well spotted